api/definition/swagger.yaml

```
@@ -1,26 +1,26 @@
 openapi: 3.0.0
 servers:
-  - description: SwaggerHub API Auto Mocking
-    url: https://virtserver.swaggerhub.com/ANDREWJEVANS/fingertips-next-api/1.0.0
+  # Added by API Auto Mocking Plugin
+  - description: API server
+    url: https://fingertips-next-api/1.0.0
 info:
   description: An API to query public health indicator data.
   version: "1.0.0"
   title: Fingertips API
   contact:
-    email: you@your-company.com
+    email:  ProfileFeedback@dhsc.gov.uk
```

Think there's an extra space here:
```suggestion
    email: ProfileFeedback@dhsc.gov.uk
```